x-pack/plugins/ml/common/util/job_utils.ts

```diff
@@ -99,6 +109,31 @@ export function isSourceDataChartableForDetector(job: CombinedJob, detectorIndex
         scriptFields.indexOf(dtr.by_field_name!) === -1 &&
         scriptFields.indexOf(dtr.over_field_name!) === -1;
     }
+
+    // We don't currently support nested terms aggregation
+    // & when aggregation interval is different from bucket span
+    const hasDatafeed =
+      typeof job.datafeed_config === 'object' && Object.keys(job.datafeed_config).length > 0;
+    if (hasDatafeed) {
+      const aggs = getDatafeedAggregations(job.datafeed_config);
+      if (aggs !== undefined) {
+        const aggBucketsName = getAggregationBucketsName(aggs);
+        if (aggBucketsName !== undefined) {
+          // if datafeed has any nested terms aggregations at all
+          const aggregations = getAggregations<{ [key: string]: any }>(aggs[aggBucketsName]) ?? {};
+          const termsField = findAggField(aggregations, 'terms', true);
+          if (termsField !== undefined && Object.keys(termsField).length > 1) {
+            return false;
+          }
+
+          // if aggregation interval is different from bucket span
+          const datetimeBucket = aggs[aggBucketsName].date_histogram;
```

Comment:
For this use case, I think we should enable access to the Single Metric Viewer, and show the charts in the Anomaly Explorer, but display a Toast / Callout which explains that the numbers on the chart may differ from the values reported for the anomaly. For example, this chart provides a lot of value:

![image](https://user-images.githubusercontent.com/7405507/100769484-0131a680-33f4-11eb-906e-50dfa2a824dd.png)

Reply:
Update - as discussed, we should just show the charts in this case, with no Toast / Callout about the differing bucket span and aggregation interval, as otherwise they would show up for jobs created in the Single Metric wizard, where we set the agg interval to 10% of the bucket span. We can address that edge case in a future PR, to show a toast warning only if the `function` used in the detector is different to that used in the datafeed aggregation.